src/styles/app.css

```diff
@@ -36,3 +50,25 @@ h2 a.tutorial-header-link {
 h2:hover a.tutorial-header-link {
   text-decoration: underline;
 }
+
+.qpp-docs-title-container {
+	background: #036;
```

Comment:
Yeah seems like a good idea! I don't know much about sass but I know that a lot of people use it